File: PRNumberBiz/app/src/main/java/com/pplus/prnumberbiz/apps/billing/data/BillingData.java
package com.pplus.prnumberbiz.apps.billing.data;

import android.os.Parcel;
import android.os.Parcelable;

/**
 * Created by imac on 2017. 11. 13..
 */

public class BillingData implements Parcelable{
    private String amount;
    private String point;
    private String id;

    public BillingData(String amount, String id){

        this.amount = amount;
        this.id = id;
    }

    public BillingData(String amount, String point, String id){

        this.amount = amount;
        this.point = point;
        this.id = id;
    }

    public String getAmount(){

        return amount;
    }

    public void setAmount(String amount){

        this.amount = amount;
    }

    public String getId(){

        return id;
    }

    public void setId(String id){

        this.id = id;
    }

    public String getPoint(){

        return point;
    }

    public void setPoint(String point){

        this.point = point;
    }

    @Override
    public String toString(){

        return "BillingData{" + "amount='" + amount + '\'' + ", point='" + point + '\'' + ", id='" + id + '\'' + '}';
    }


    @Override
    public int describeContents(){

        return 0;
    }

    @Override
    public void writeToParcel(Parcel dest, int flags){

        dest.writeString(this.amount);
        dest.writeString(this.point);
        dest.writeString(this.id);
    }

    protected BillingData(Parcel in){

        this.amount = in.readString();
        this.point = in.readString();
        this.id = in.readString();
    }

    public static final Creator<BillingData> CREATOR = new Creator<BillingData>(){

        @Override
        public BillingData createFromParcel(Parcel source){

            return new BillingData(source);
        }

        @Override
        public BillingData[] newArray(int size){

            return new BillingData[size];
        }
    };
}

File: PRNumber/app/src/main/java/com/pplus/prnumberuser/core/network/model/dto/BillingData.kt
package com.pplus.prnumberuser.core.network.model.dto

import android.os.Parcel
import android.os.Parcelable
import com.google.gson.JsonObject
import com.google.gson.JsonParser
import java.util.*

/**
 * Created by imac on 2018. 1. 2..
 */
class BillingData(var amount: String? = null,
                  var point: String? = null,
                  var id: String? = null) : Parcelable {
    constructor(source: Parcel) : this(
            source.readString(),
            source.readString(),
            source.readString()
    )

    override fun describeContents() = 0

    override fun writeToParcel(dest: Parcel, flags: Int) = with(dest) {
        writeString(amount)
        writeString(point)
        writeString(id)
    }

    companion object {
        @JvmField
        val CREATOR: Parcelable.Creator<BillingData> = object : Parcelable.Creator<BillingData> {
            override fun createFromParcel(source: Parcel): BillingData = BillingData(source)
            override fun newArray(size: Int): Array<BillingData?> = arrayOfNulls(size)
        }
    }
}